helm-charts/helm3/strimzi-kafka-operator/crds/040-Crd-kafka.yaml

```diff
@@ -1,15 +1,10 @@
-{{- if .Values.createGlobalResources -}}
```

Comment:
All templating is removed from the CRDs as templates are not support in the `crds` directory for Helm 3.